src/params.rs

```diff
@@ -35,6 +35,9 @@ pub struct Params {
 
 	/// Arguments to forward to the kernel
 	pub kernel_args: Vec<String>,
+
+	/// Paths that should be mounted on-device.
+	pub mount: Option<Vec<String>>,
```

Comment:
`mount` is maybe not the best name here. What about `file_map` or `file_access_map`?

Reply:
https://github.com/n0toose/uhyve/commit/e7079f9ad6eb2a66e5493a5c19eaef073bb3bccd https://github.com/n0toose/uhyve/commit/9a59a93bd77755e39b9b727a0117f8a338a28a04